.ai-bot/instructions.md

```
@@ -0,0 +1,228 @@
+
+  locales/           ← i18n translation files (en.json + others)
+  types/             ← @openshift-assisted/types package (shared type definitions)
+```
+
+---
+
+## Wizard Step Order (OCM)
+
+Defined in `libs/ui-lib/lib/ocm/components/clusterWizard/wizardTransition.ts`:
+
+```
```

markdownlint-cli2 (0.22.1)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)